cdxev/__main__.py

```diff
@@ -447,31 +447,22 @@ def create_validation_parser(
     group = parser.add_mutually_exclusive_group()
     group.add_argument(
         "--schema-type",
-        help=(
-            "Decide whether to use the default specification of CycloneDX or a custom schema. "
-            "The version will be derived from the specVersion in the provided SBOM. "
-            "If no version is provided defaults to 1.3."
-        ),
+        help="Use a built-in schema for validation.",
```

Comment:
Why did you remove the information that the version is derived from `specVersion`. Now this information is only within the documentation, which is kind of against your wish that persons will only rely on `--help` and also the generation with sphinx won't work then.